lib/internal/webstreams/queuingstrategies.js

```diff
@@ -47,11 +48,12 @@ const isCountQueuingStrategy =
  * }} QueuingStrategy
  */
 
-// eslint-disable-next-line func-name-matching,func-style
-const byteSizeFunction = function size(chunk) { return chunk.byteLength; };
+const byteSizeFunction = (chunk) => { return chunk.byteLength; };
+const countSizeFunction = () => { return 1; };
 
-// eslint-disable-next-line func-name-matching,func-style
-const countSizeFunction = function size() { return 1; };
+for (const fn of [byteSizeFunction, countSizeFunction]) {
+  ObjectDefineProperty(fn, 'name', { __proto__: null, value: 'size' });
+}
```

Comment:
Using for..of on an array relies on user-mutable `Array.prototype[Symbol.iterator]`.
```suggestion
```

Reply:
Thanks. Applied the suggestion.